docs/user/openstack/known-issues.md

```diff
@@ -0,0 +1,15 @@
+# Known Issues and Workarounds
+
+During this release, we are shipping with a few known issues. We are documenting them here, along with whatever workarounds we are aware of, and have attached the links to where the engineering team is tracking the issues. As changes occur, we will update both this document, and the issue trackers with the lates information.
+
+## Long Hostnames
+
+The cluster sets the hostname of servers to `<clusterName> Workstation-<clusterName>-<clusterID>-master-<#>`. There is a character limit of 64, so creating a cluster name that is too long could result in an installer crash. In order to prevent this from happening, please restrict the length of the `metadata.clusterName` field to 18 characters as a baseline. This value is dependant on the number of masters you have, since the masters are numbered. For each multiple of 10, subtract 1 from the max number of characters. For example, if you have 25 masters, your cluster name will be restricted to 17 characters. The installer will validate this in your install config and throw an error to prevent you from triggering this install time bug. This is being tracked in this [github issue](https://github.com/openshift/installer/issues/2243).
+
+## Network Names
+
+Since the installer requires the *Name* of your external network, if you have other networks with the same name, it will choose one randomly. This is not a reliable way to run the installer. We highly recommend that you resolve this with your cluster administrator by creating unique names for your networks.
```

Comment:
We should also document that the interactive prompt is not an option when your external network CIDR overlaps one of the default network ranges. In which case, you need to provide the installer with an `install-config.yaml` file.